### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Websites_Keyboard: While navigating inside the expanded "More" menu items, keyboard focus is moving to the background controls of the page after the last menu item and hence screen reader is reading the same.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#104]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106e
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS2.4.3, MAS1.3.2

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Verify that keyboard focus is proper while navigating inside the "More" menu items.

**Actual Result:**
While navigating inside the expanded "More" menu items, keyboard focus is moving to the background controls of the page after the last menu item and hence screen reader is reading the same.

**Expected Result:**
When keyboard focus should not move out of the expanded "More" menu items. When user try to press "Tab/arrow" key after last control, keyboard focus should not move ahead. Here screen reader should alert user by announcing it as last menu item. 

**Note**: This issue is observed throughout the application.

**User Impact:**
Users will find it difficult to properly navigate throughout the page if keyboard focus will not be proper. Screen reader users will find it difficult to understand the navigation if it reads the background contents after the last menu item.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment:**
[Bug 104_focus moving out of more dropdown menu.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323659/Bug.104_focus.moving.out.of.more.dropdown.menu.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
